UnityProject/Assets/Scripts/UI/Core/Net/Elements/NetClientSyncString.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UI.Core.NetUI;
+using UnityEngine;
+
+public class NetClientSyncString : NetServerSyncString
```

Comment:
why is this a whole class in an entire script?

Reply:
makes it a bit more clear?